examples/physics_informed_fitness.ipynb

```diff
@@ -0,0 +1,411 @@
+{
```

Comment:
This is a hard example to automate, it doesn't seem feasible to get a good model on this problem in a short period of time. Ideally the example runs in a short period of time so it can be tested quickly.

Suggestions are welcomed here. See [the doc page](https://nightdr.github.io/bingo/_low_level/physics_fitness.html) for more details.